spec/feature/post_system_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :system do
```

Comment:
- [ ] Hi @SekoViper, most of your test cases are failing as you can see on the screenshot below.

![Screenshot from 2023-03-22 13-04-05](https://user-images.githubusercontent.com/63087491/226900360-748f3fef-30eb-4679-a880-c34c6948290f.png)